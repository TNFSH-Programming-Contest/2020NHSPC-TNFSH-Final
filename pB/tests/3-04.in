76
249.4 -263.2
227.994 -126.328
133.222 -56.829
-18.413 -85.26
-65.798 -161.078
-27.89 -221.1
25.814 -265.327
87.416 -249.531
95.314 -192.668
57.0 -167.0
49.507 -224.259
-8.935 -175.293
47.928 -104.215
171.131 -145.282
155.336 -304.815
-37.367 -325.349
-132.139 -224.259
-95.81 -34.715
104.791 15.83
278.539 -91.578
305.391 -274.804
240.631 -405.905
-1.038 -456.45
-201.638 -407.485
-277.456 -214.782
-231.6 -70.8
-297.99 7.932
-196.9 3.194
-152.673 109.022
-98.969 97.965
130.063 110.602
297.494 61.636
366.993 -50.51
441.8 -167.0
411.22 -266.906
411.22 -450.132
216.938 -510.154
-5.776 -527.529
-133.718 -499.097
-231.6 -551.8
-313.785 -506.995
-291.671 -462.768
-376.966 -192.668
-365.909 47.42
-299.569 140.613
57.0 314.0
-369.068 202.214
-424.0 -167.0
-346.955 -420.121
-328.001 -609.665
-135.4 -551.8
107.95 -603.346
333.823 -513.313
398.584 -527.529
510.73 -446.973
457.026 -366.417
509.151 -224.259
330.664 115.34
-8.935 145.351
-179.525 137.454
-201.638 52.159
-332.739 41.102
-309.046 -83.681
-326.421 -198.986
-252.183 -451.711
40.03 -495.938
305.391 -426.439
363.834 -167.0
288.017 -4.704
126.904 66.375
-154.252 55.318
-209.536 -118.43
-182.684 -268.486
-95.81 -393.269
10.019 -413.803
183.767 -375.894
73.835 -204.518
