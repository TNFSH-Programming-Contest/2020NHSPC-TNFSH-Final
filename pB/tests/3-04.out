1135981.014